qiskit_nature/converters/second_quantization/qubit_converter.py

```diff
@@ -369,19 +396,17 @@ def _symmetry_reduce(
             symmetry_ops = []
             for symmetry in self._z2symmetries.symmetries:
                 symmetry_ops.append(PauliSumOp.from_list([(symmetry.to_label(), 1.0)]))
-            commuted = []
-            for i, qubit_op in enumerate(qubit_ops):
+            commuted = {}
+            for name, qubit_op in iter(qubit_ops):
                 commutes = QubitConverter._check_commutes(symmetry_ops, qubit_op)
-                commuted.append(commutes)
+                commuted[name] = commutes
                 logger.debug("Qubit operators commuted with symmetry %s", commuted)
```

Comment:
Once we start to have names here as keys then it might be nice to improve the debug message to include the operator name maybe instead of keep outputting the commuted object